config/routes.rb

```diff
@@ -53,4 +53,8 @@
 
   match "/api", via: [:get], to: redirect("/api/v1")
   root "podcasts#index"
+
+  unless Rails.configuration.consider_all_requests_local
+    match "*unmatched", via: :all, to: "errors#not_found"
```

Comment:
For local dev, I left things alone.  You'll still get a log.error and the fancy "route missing" page.  But in prod, they just get the static 404 page.